bindata/etcd/cluster-backup.sh

```diff
@@ -60,23 +35,27 @@ function source_required_dependency {
   source "${path}"
 }
 
-BACKUP_DIR="$1"
-DATESTRING=$(date "+%F_%H%M%S")
-BACKUP_TAR_FILE=${BACKUP_DIR}/static_kuberesources_${DATESTRING}.tar.gz
-SNAPSHOT_FILE="${BACKUP_DIR}/snapshot_${DATESTRING}.db"
-BACKUP_RESOURCE_LIST=("kube-apiserver-pod" "kube-controller-manager-pod" "kube-scheduler-pod" "etcd-pod")
 
-trap 'rm -f ${BACKUP_TAR_FILE} ${SNAPSHOT_FILE}' ERR
+trap 'podman rm --force cluster-backup-$$' ERR
 
 source_required_dependency /etc/kubernetes/static-pod-resources/etcd-certs/configmaps/etcd-scripts/etcd.env
```

Comment:
we had a bug on this before that we need to check if this exists before we source.

Reply:
That is what source_required_dependency does.

Reply:
ok sry miseed that